AcadeMeData/tests.py

```diff
@@ -1,3 +1,39 @@
-# from django.test import TestCase
-
-# Create your tests here.
+import pytest
+from AcadeMeData.models import User
+
+
+class TestUserModel(pytest):

```

Comment:
test_get_user -> creates user and test the deletion
test_del_user -> creates user and test the deletion

I think you can structurize it better and put the common part (creates_user) in its own function. There is no reason for test_get_user to test user deletion

Reply:
SOLVED